loadgen/ebbandflow/ebb_and_flow.go

```diff
@@ -0,0 +1,21 @@
+package ebbandflow
+
+import (
+	"time"
+
+	"github.com/temporalio/omes/loadgen"
+)
+
+type WorkflowParams struct {
+	SleepActivities *loadgen.SleepActivityConfig `json:"sleepActivities"`
+}
+
+type WorkflowOutput struct {
+	Timings []ActivityTiming `json:"timings"`
+}
+
+type ActivityTiming struct {
+	FairnessKey       string        `json:"fairnessKey"`
+	FairnessWeight    float32       `json:"fairnessWeight"`
+	ScheduleToStartMS time.Duration `json:"scheduleToStartMS"`
```

Comment:
it's a duration?
```suggestion
	ScheduleToStart time.Duration `json:"scheduleToStart"`
```
also maybe make these json keys shorter, e.g. single char, since there's going to be a big array of them?